examples/swift/hello_world/ViewController.swift

```diff
@@ -65,7 +65,13 @@ final class ViewController: UITableViewController {
         NSLog("Response data (\(requestID)): \(data.count) bytes")
       }
       .onError { [weak self] error in
-        let message = "failed within Envoy library: \(error.message)"
+        var message: String
```

Comment:
```suggestion
        let message: String
```
Compiler enforces that this is set exactly once here

Reply:
I wonder why my local compilation worked?

Reply:
It'll compile, just doesn't need to be mutable